### PR TITLE
Document for SBO 1.0.1 Release notes

### PR DESCRIPTION
**Pull request by Srivaralakshmi**
- **Aligned team**: Dev Tools
- **OCP version for cherry-picking**: enterprise-`4.9`, enterprise-`4.10`
- **JIRA issues**: [RHDEVDOCS-3572](https://issues.redhat.com/browse/RHDEVDOCS-3572)
- **Preview pages**: [Release notes for Service Binding Operator](https://deploy-preview-42134--osdocs.netlify.app/openshift-enterprise/latest/applications/connecting_applications_to_services/sbo-release-notes.html)
- **SME Review**: Approved by @baijum, @dperaza4dustbit 
- **QE review**:  Approved by @pmacik 
- **Peer-review**: Approved by @jc-berger

### CODE REVIEW

**Review by dperaza4dustbit — COMMENTED** [review]
lgtm

File: modules/sbo-release-notes-1-0-1.adoc [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+* Before this update, when you bind a `PostgresCluster` CR of the `postgres-operator.crunchydata.com/v1beta1` API, the binding data values did not include the values for the database certificates. As a result, the application failed to connect to the database. With this update, modifications to the annotations that the {servicebinding-title} uses to expose the binding data from the backing service CR now include the database certificates. The {servicebinding-title} uses these modified annotations to project the correct `ca.crt`, `tls.crt`, and `tls.key` certificate files. link:https://issues.redhat.com/browse/APPSVC-1045[APPSVC-1045]
```
Comment (jc-berger) [inline modules/sbo-release-notes-1-0-1.adoc:52]: Verb usage: `when you _would_ bind a PostgresCluster...the binding data values did not include...`

File: modules/sbo-release-notes-1-0-1.adoc [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+* Before this update, when you bind a `PerconaXtraDBCluster` custom resource (CR) of the `pxc.percona.com` API, the binding data values did not include the `port` and `database` values. These binding values along with the others already projected are necessary for an application to successfully connect to the database service. With this update, the annotations that the {servicebinding-title} uses to expose the binding data values from the backing service CR are now modified to project the additional `port` and `database` binding values. The {servicebinding-title} uses these modified annotations to project the complete set of binding values that the application can use to successfully connect to the database service. link:https://issues.redhat.com/browse/APPSVC-1073[APPSVC-1073]
```
Comment (jc-berger) [inline modules/sbo-release-notes-1-0-1.adoc:53]: Verb usage: `when you _would_ bind a PerconaXtraDBCluster...the binding data values did not include the port...`

**Review by jc-berger — APPROVED** [review]
/lgtm

**Comment by Srivaralakshmi** [comment]
Hi @jc-berger: Thanks for the suggestions. I have addressed them. PTAL and provide your LGTM.

**Comment by Srivaralakshmi** [comment]
Hi, @pmacik Can you PTAL and provide your approval? Thanks!

File: modules/sbo-release-notes-1-0-1.adoc [hunk 3]
```diff
@@ -0,0 +1,87 @@
+[id="sbo-release-notes-1-0-1_{context}"]
+= Release notes for {servicebinding-title} 1.0.1
+
+{servicebinding-title} is now available on {product-title} 4.7, 4.8 and 4.9.
+
+{servicebinding-title} 1.0.1 supports OpenShift Container Platform 4.9 and later running on:
+
+* IBM Power Systems
+* IBM Z and LinuxONE
```
[the remark this answers is not in the document]
Comment (Srivaralakshmi) [reply modules/sbo-release-notes-1-0-1.adoc:9]: SBO runs on x86_64 platform from the very beginning. So, there is no need to mention this separately.

**Review by jc-berger — APPROVED** [review]
/lgtm

**Pull request merged**